persistent_stream.go

```diff
@@ -274,26 +276,18 @@ func (d *Daemon) getPersistentStreamHandler(cw ggio.Writer) network.StreamHandle
 	}
 }
 
-func (d *Daemon) sendReponseToRemote(req *pb.PersistentConnectionRequest) *pb.PersistentConnectionResponse {
+func (d *Daemon) sendReponseToRemote(req *pb.PersistentConnectionRequest) {
 	callID, err := uuid.FromBytes(req.CallId)
 	if err != nil {
-		return errorUnaryCallString(
-			callID,
-			"malformed request: call id not in UUID format",
-		)
+		return
```

Comment:
I suggest to:

(1) Either log the error here with a low loglevel
(2) Or return the error in `sendReponseToRemote()` and log it with a low loglevel in the caller (`handlePersistentConnRequest()`)